### PR TITLE
Stylefix

### PR DESCRIPTION
**Pull request by spbyrne**
* Creates blog pagination component, adds styling
* Improve font fallback behaviour
* Add react memo to docs layout & header nav
* Adjust various styles
* Add iframe styles to richtext
* Closes docs section on nav (if no longer in that section)
* Reduce homepage code size on mobile

### CODE REVIEW

File: components/ui/BlogPagination.tsx [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+    const prevPage =
+      currentPage - 1 === 1 ? '/blog' : `/blog/page/${currentPage - 1}`
+    const nextPage = `/blog/page/${currentPage + 1}`
+    const [selectValue, setSelectValue] = useState(currentPage)
+
+    function handleSelectChange(e) {
+      e.preventDefault()
+      const pageNumber = e.target.value
+      setSelectValue(pageNumber)
+      if (pageNumber === '1') {
```
Comment (jamespohalloran) [inline components/ui/BlogPagination.tsx]: I don't think we need this conditional, as /blog/page/1 should work

File: components/ui/BlogPagination.tsx [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+import Router from 'next/router'
+
+import { DynamicLink } from './DynamicLink'
+
+export const BlogPagination = styled(
+  ({ currentPage, numPages, ...styleProps }) => {
+    const isFirst = currentPage === 1
+    const isLast = currentPage === numPages
+    const prevPage =
+      currentPage - 1 === 1 ? '/blog' : `/blog/page/${currentPage - 1}`
```
Comment (jamespohalloran) [inline components/ui/BlogPagination.tsx]:
```suggestion
      `/blog/page/${currentPage - 1}`
```
/blog/page/1 should still work

File: components/ui/BlogPagination.tsx (hunk 1 above)
Comment (spbyrne) [reply components/ui/BlogPagination.tsx]: Tested and indeed that works, but I do like that this keeps the URL clean (/blog vs. /blog/page/1). I'm fine with removing it, though. Simplifies things.

Comment (spbyrne) [reply components/ui/BlogPagination.tsx]: @kendallstrautman What do you think?

Comment (kendallstrautman) [reply components/ui/BlogPagination.tsx]: yeah that's fine as long as `/blog` still works

Comment (kendallstrautman) [reply components/ui/BlogPagination.tsx]: ill fix this in another branch

**Pull request merged**